### PR TITLE
Meal endpoint

### PR DESCRIPTION
**Pull request by soniasonia**

### CODE REVIEW

File: backend/meal/models.py [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+        on_delete=models.CASCADE,
+    )
+    ingredients = models.ManyToManyField('Ingredient', through='MealIngredient')
+
+    @property
+    def total_kcal(self):
+        meal_ingredients = MealIngredient.objects.filter(meal=self)
+        if len(meal_ingredients) > 0:
+            return sum(i.kcal for i in meal_ingredients)
+        else:
```
Comment (m4r3x) [inline backend/meal/models.py]: no need for else

File: backend/tests/api/test_manage_user.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    def setUp(self):
+        self.client = APIClient()
+        self.testTables = [
+            {
+                'name': 'Test that unauthorized user cannot retrieve profile',
+                'request': (lambda: self.client.get(ME_URL)),
+                'expected_status_code': status.HTTP_401_UNAUTHORIZED,
+            },
+            {
+                'name': 'Test that unauthorized user cannot POST profile',
```
Comment (m4r3x) [inline backend/tests/api/test_manage_user.py]: isn't it exactly the same test as per line 31? same endpoint, same expected status, just different name. Please remove one.

**Review by m4r3x — APPROVED** [review]
just minor things, please format the code properly and feel free to merge to master.

I also noticed you added some frontend, but I do not see your code review at: https://github.com/soniasonia/meal-tracker/pull/4

File: backend/meal/models.py (hunk 1 above)
Comment (soniasonia) [reply backend/meal/models.py]: OK

File: backend/tests/api/test_manage_user.py (hunk 2 above)
Comment (soniasonia) [reply backend/tests/api/test_manage_user.py]: OK

**Pull request merged**